CuPy installation found, continuing using GPU acceleration.
having the CPUs {0, 2, 4} available.
Started basinhopping at 2021-01-16 21:05:17.586557 UTC, with
command line arguments = {'path': '/home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18', 'n_par': 26, 'par_multiplicity': 9, 'n_iter': 0, 'cost_fn': 'energy', 'temperature': 1.0, 'stepsize': 1.0, 'init_par': None, 'dump_interval': None}
runtime variables =  {'GPU': True, 'date_start': '2021-01-16 21:05:17.586557', 'complete_graph': [(1, 2), (3, 4), (5, 6), (7, 8), (9, 10), (11, 12), (13, 14), (15, 16), (17, 0), (0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)], 'init_layer': [(0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)], 'layers': [[(1, 2), (3, 4), (5, 6), (7, 8), (9, 10), (11, 12), (13, 14), (15, 16), (17, 0)], [(0, 1), (2, 3), (4, 5), (6, 7), (8, 9), (10, 11), (12, 13), (14, 15), (16, 17)]], 'n': 18}
.............................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................../home/jkgpu/miniconda3/lib/python3.8/site-packages/chainer/backends/cuda.py:142: UserWarning: cuDNN is not enabled.
Please reinstall CuPy after you install cudnn
(see https://docs-cupy.chainer.org/en/stable/install.html#install-cudnn).
  warnings.warn(
/home/jkgpu/miniconda3/lib/python3.8/site-packages/chainer/_environment_check.py:70: UserWarning: 
--------------------------------------------------------------------------------
CuPy (cupy) version 8.0.0 may not be compatible with this version of Chainer.
Please consider installing the supported version by running:
  $ pip install 'cupy>=6.3.0,<7.0.0'

See the following page for more details:
  https://docs-cupy.chainer.org/en/latest/install.html
--------------------------------------------------------------------------------

  warnings.warn(msg.format(
 
==========================================================================================
Finished basinhopping of  /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18 at 2021-01-16 22:13:58.727632 UTC, with
 
{'path': '/home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18', 'n_par': 26, 'par_multiplicity': 9, 'n_iter': 0, 'cost_fn': 'energy', 'temperature': 1.0, 'stepsize': 1.0, 'init_par': None, 'dump_interval': None}
init_par = [-6.529292339485851e-05, 0.0001162543879128031, -0.00022890919999998105, -5.389869020179164e-05, 7.337522703175124e-05, -1.2200598278353317e-05, -0.00017740711577596534, -3.851780451246512e-05, -0.0004987213125551041, -0.00022982267407864465, -7.357271634944942e-05, -0.0001476256509374576, 0.0002568719051913641, -0.0002631983432283642, -0.00024597632557601544, 9.773324888374252e-05, -0.0003721016054787971, -5.273146725283742e-05, 9.398250907260205e-05, -0.00032163337671673787, 0.00027430669375854864, 0.00031430215341880167, 0.00010010018141156957, 5.4959911415881175e-05, 3.147320914238989e-05, -0.00046460127266102216]
 
RESULTS:
--------
E_VQE = -7.987451326678516
inf_VQE = 0.045054914701414206
n_fn_calls = 1039
Wall-clock time 1.1445306240850026 hours
 
For more output, see /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18/output.txt
Plots of output saved in /home/jkgpu/HVQE/data/chain/periodic/own_graph/ops/energy/18
End of HVQE.py
==========================================================================================
